pass-bakery/build.sbt

```diff
@@ -1,3 +1,6 @@
+import scala.Console.in
```

Comment:
Can you explain why you needed this?

Reply:
Not sure why I thought I did, not currently using it. I've removed this.